Wikipedia/Code/WMFImageURLParsing.m

```diff
@@ -6,7 +7,7 @@
     dispatch_once(&onceToken, ^{
         // TODO: try to read serialized regex from disk to prevent needless pattern compilation on next app run
         NSError* patternCompilationError;
-        imageNameFromURLRegex = [NSRegularExpression regularExpressionWithPattern:@"^\\d+px-([^.]*(\\.[^.$]{4,})*\\.[^.]*).*$"
+        imageNameFromURLRegex = [NSRegularExpression regularExpressionWithPattern:@"^\\d+px-(.*)"
```

Comment:
@bgerstle I really appreciate the heads up! I think this is working? The lead image opens and shares fine.